package.json

```diff
@@ -22,8 +22,8 @@
     "bootstrap": "git submodule update --init && cd eslint && npm install",
     "eslint": "cd eslint && mocha -c tests/lib/rules/*.js -r ../eslint-tester.js",
     "test": "mocha",
-    "lint": "./node_modules/eslint/bin/eslint.js index.js babylon-to-espree test",
-    "fix": "./node_modules/eslint/bin/eslint.js index.js babylon-to-espree test --fix",
+    "lint": "node node_modules/eslint/bin/eslint.js index.js babylon-to-espree test",
+    "fix": "node node_modules/eslint/bin/eslint.js index.js babylon-to-espree test --fix",
```

Comment:
Shouldn't these just rely on npm to set the path and be `eslint index.js babylon-to-espree test`?